dcos/http.py

```diff
@@ -27,6 +27,34 @@ def _default_is_success(status_code):
 
     return 200 <= status_code < 300
 
+def _is_request_to_dcos(url, toml_config=None):
+    """Checks if a request is for the DC/OS cluster.
+
+    :param url: URL of the request
+    :type url: str
+    :param toml_config: cluster config to use
+    :type toml_config: Toml
+    :return: whether the request is for the cluster
+    :rtype: bool
+    """
+
+    if toml_config is None:
+        toml_config = config.get_config()
+
+    dcos_url = urlparse(config.get_config_val("core.dcos_url", toml_config))
+    cosmos_url = urlparse(
```

Comment:
I also don't have it set, in that case it is ignored, urlparse will return an empty ParseResult and this won't be matched.